### PR TITLE
Always check issuer and audience

### PR DESCRIPTION
**Pull request by kjellkongsvik**
This PR prepares for sending storage endpoint url to core and using JWT to authenticate and authorize to the storage

### CODE REVIEW

File: api/server/sliceModel.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type slicer struct {
-	root string
+	root     string
```
Comment (sveinung-r) [inline api/server/sliceModel.go:12]: This commit only adds whitespace. Does not match with the title.

Comment (kjellkongsvik) [reply api/server/sliceModel.go]: agreed. cleaned alot

File: docker-compose_tests.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@ services:
     environment:
       - AUTHSERVER=http://auth:8089/common
       - ISSUER=auth:8089
+      - AUDIENCE=test
```
Comment (sveinung-r) [inline docker-compose_tests.yml:17]: This part belongs in the check audience commit

Comment (kjellkongsvik) [reply docker-compose_tests.yml:17]: true, same with change in integration_tests.py

**Pull request merged**